CHANGELOG.md

```diff
@@ -4,10 +4,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org/).
 
+[1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.1.0] - 2023-09-26
+
+### Changed
+
+- Change the streamed amount card in NFT descriptor with deposit amount
```

Comment:
Let's say:

"Replace the streamed amount with the deposit amount in the NFT descriptor"